### PR TITLE
Updates for trac ticket #389 (Fix comparisons to None in step code)

### PR DESCRIPTION
**Pull request by dmggh**
@hbushouse None comparison updates for multiple steps

### CODE REVIEW

File: jwst/saturation/saturation.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from . import x_irs2
 
 log = logging.getLogger(__name__)
-log.setLevel(logging.DEBUG)
+log.setLevel(logging.INFO)
```
Comment (hbushouse) [inline jwst/saturation/saturation.py]: Want to keep this set to DEBUG, so that debug-level messages will be passed when requested.

**Comment by hbushouse** [comment]
Other than the one instance of logging level that got reverted from a recent update, the rest looks good. 👍

**Comment by hbushouse** [comment]
@dmggh you should get the conflicts resolved and get these old commits merged in, before they get even more out of date.

**Comment by dmggh** [comment]
Rebasing and pushing to origin appears to have merged the correct files (but did not result in a new pull request), so closing this pull request.

**Pull request closed without merging**